tests/analyzer/test_condition.py

```diff
@@ -41,6 +41,40 @@ def test_check():
     row = {(CONDITION_FLAG, ("public.patients", "age")): 3}
     assert cond.check(row)
 
+    # IN integers
```

Comment:
do you think it's worth testing dates? dates should not contain commas anyways so I'm not sure